zimui/src/utils/parseChallenge.ts

```diff
@@ -4,6 +4,10 @@ export const parseChallenge = (markdownStr: string): Challenge => {
   return new Challenge(marked.lexer(markdownStr))
 }
 
+export const extractKeyValueFromMarkdown = (rawString: string): string[] => {
```

Comment:
Please add a comment explaining what you're doing and why. This returns a tuple of topic and title

Reply:
Not really topic and title, it is a tuple key and value for a given markdown header. Anyway, comment is now making this clearer (hopefully ^^)